src/organisms/forms/GeneralForm/index.js

```diff
@@ -8,19 +8,11 @@ function GeneralForm( props ) {
   const {
     className,
     children,
-    formName,
-    onSubmit,
-
   } = props;
 
   return (
-    <div className={classnames( styles[''], className )}>
-      <form
-        onSubmit={onSubmit}
-        name={formName}
-      >
-        { children }
-      </form>
+    <div className={classnames( styles['general-form'], className )}>
+      { children }
```

Comment:
Alternatively, can't you just pass in the form props on `<GeneralForm />` and spread them on them on `<form />`  with `...rest`